config.go

```diff
@@ -90,6 +90,13 @@ const (
 	// hours i.e. the cleanup will happen every day.
 	DefaultStaleDataCleanupInterval = 24 * time.Hour
 
+	// DefaultQueryMissionControlBatchSize specifies the default number of
+	// pairs to be sent in each batch when querying the aggregated mission
+	// control data. The size of a given mission control pair is ~114 bytes
+	// as defined in the proto file. With the default value of 4600 pairs,
+	// the batch size would be approximately 512 KB (1/2 MB).
```

Comment:
It is not mainly a bottleneck to the REST wrapper (but maybe this is a side benefit?). Mainly I used this as an optimization technique to avoid large message transmission and higher memory consumption.

Updated the comments!